documentation/src/main/resources/_posts/2021-09-23-conditional-requests.md

```diff
@@ -102,7 +103,8 @@ it was **lastModified** after "2021-08-25T12:38:27".
 
 ### Permissions to execute the example
 For this example, the authorized subject could have READ and WRITE permissions on the complete thing resource.
-However, it is only necessary on the path _thing/features/water-tank/status_, because the **temperature** as well as the conditional part **lastModified** are located there.
+However, it is only necessary on the path _thing/features/water-tank/status_, because the **temperature** as well as 
```

Comment:
Is here a colon missing? `_thing:/features/water-tank/status`

Reply:
Good catch. I will correct it.

Reply:
Oh, and 'properties' is missing from that resource path.

Reply:
Also fixed the resource path ;)